Aspire BudgetingTests/DashboardGroupsAndCategoriesTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DashboardGroupsAndCategoriesTests.swift
+//  Aspire BudgetingTests
+//
+//  Created by TeraMo Labs on 11/17/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+@testable import Aspire_Budgeting
+import XCTest
+
+class DashboardGroupsAndCategoriesTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/2e101a0070745b5b54cd4d58f3d54a7c1a52654c/Aspire BudgetingTests/DashboardGroupsAndCategoriesTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>